Backend/mentii/class_ctrl.py

```diff
@@ -116,7 +117,9 @@ def getClassCodesFromUser(dynamoDBInstance, email=None):
     request = {"Key" : {"email": email}, "ProjectionExpression": "classCodes"}
     res = dbUtils.getItem(request, usersTable)
     #Get the class codes for the user.
-    if res is not None and 'Item' in res:
+    if res is None or 'Item' not in res or 'classCodes' not in res['Item']:
+      MentiiLogging.getLogger().error('Unable to get user data in getClassCodesFromUser')
+    else:
       classCodes = res['Item']['classCodes']
   return classCodes
```

Comment:
One thing to note, we initialize classCodes as a list at the top of the method and then return what I assume is a set from the res['Item'] dictionary. We should probably initialize classCodes for consistency and to know that this function is going to return a set no matter what.